src/containers/GeetestCaptcha/index.tsx

```diff
@@ -3,7 +3,8 @@ import {
     connect,
     MapStateToProps,
 } from 'react-redux';
-import { initGeetest } from '../../helpers/geetest';
+// MUST BE REMOVED: Vadym P. responsibility
```

Comment:
1. Why do we need to remove geetest captcha?
2. Why it's your responsibility?

Reply:
1. It should be rewritten due to very bad code in JavaScript. Should be replaced with some NPM package written on TypeScript.
2. Because I doing refactoring.

Reply:
will push fix for `geetest` to this PR